orderer/consensus/hlmirbft/chain.go

```diff
@@ -360,6 +360,8 @@ func (c *Chain) Start() {
 	c.Node.start(c.fresh, isJoin)
 
 	close(c.startC)
+	//JIRA FLY2-98 close the error channel
+	close(c.errorC)
```

Comment:
The closing of `c.errorC` marks the occurrence of an error not the absence of errors. It shouldn't be closed here, but instead within the go routines in `node.go` that initiate the Mir state machine, in the event of an error. Specifically the go routine that holds`ProcessAsNewNode` and the go routine that holds `RestartProcessing`.